site/en/docs/webstore/discovery/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: "layouts/doc-post.njk"
+title: Discovery on the Chrome Web Store
+date: 2022-03-21
+#updated: 2022-03-21
+description: >
+  An overview of how users find items on the Chrome Web Store, and how our editors select items to
+  feature.
+---
+
+We strive to make it easy for users to discover great items on the Chrome Web Store. A positive
+discovery experience means making it simple for our users to find the items they know and love, as
+well as new and undiscovered items. From browsing the homepage to searching by title, we want users
+to find the best item to fit their needs. Learn more about discovery on the Chrome Web Store below.
+
+## Search {: #search }
+
+Chrome Web Store search is a key tool for users to find relevant and popular items. When users
+search for an item, search returns a list of items that are ranked based on a number of criteria,
+including metadata from your item's listing page. Making sure that your store listing page is
+complete, accurate and optimized is important to ensuring your item is discoverable through search.
+Learn more about [crafting a great listing page][best-listing].
+
+## Categories {: #categories }
+
+Categories on the Chrome Web Store organize items based on their main function. Users can browse
+category pages, like Shopping, Productivity or Fun, to find relevant items to install. Developers
+select a category for their item in the Developer Dashboard, which determines which category it will
+appear under in the store. Categories are only available for extensions at this time.
+
+## Extensions Homepage {: #homepage }
+
+The homepage is where we highlight great extensions and themes that we think users will love. Users
+come to this page to find new and interesting items recommended by our editors. Chrome editors
+curate items to be featured in merchandised collections and the rotating marquee based on a variety
+of factors including performance, design, usefulness, relevancy and breadth of appeal. Items
+featured on the homepage must follow our general [best practices][best-practices] to be considered,
+but there is no set checklist developers can follow to guarantee being featured. Developers cannot
+pay to be featured on the homepage. The best thing a developer can do to increase their chances of
+being featured on the homepage is to build a useful, high-quality extension that is a joy to use.
```

Comment:
```suggestion
being featured on the home page is to build a useful, high-quality extension that is a joy to use.
```

Reply:
Just opened #2469 to track this more generally. I'll update this draft to use the two word version.